### PR TITLE
Red negative buttons (EXPOSUREAPP-13864)

### PR DESCRIPTION
**Pull request by schauersbergern**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13864)

## delete: Sie planen eine Veranstaltung? Meine QR Codes

Delete one QR-Code:

Delete all QR-Codes:

## delete: Schnelltestprofil
Currently no screenshot, but changed in code
## delete: Tagebuch - Personen und Orte

Delete one location

Delete all locations

Delete one person

Delete all persons

## deny: Kamerazugriff

This cannot be edited as this is a system dialog, but permission explanation dialog:

### CODE REVIEW

**Comment by Ein-Tim** [comment]
May we get a screenshot of this change, please?

**Comment by schauersbergern** [comment]
> May we get a screenshot of this change, please?

Sure @Ein-Tim I added screenshots: https://github.com/corona-warn-app/cwa-app-android/pull/5538#issue-1362809842

File: Corona-Warn-App/src/main/res/values-night/colors.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <color name="colorTextEmphasizedButton">#FFFFFF</color>
     <color name="colorTextSixteen">#FFFFFF</color>
     <color name="colorInputText">#BFFFFFFF</color>
-    <color name="colorTextSemanticRed">#FF919A</color>
+    <color name="colorTextSemanticRed">#ED5360</color>
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/main/res/values-night/colors.xml:26]: There are other UI elements that use this color apart from these red delete buttons. Are you sure they should have this color changed in dark mode or should it be restricted to only the dialog texts?

Comment (schauersbergern) [reply Corona-Warn-App/src/main/res/values-night/colors.xml]: Added color colorTextDeleteButtonDialog for dialogs

**Review by SamuraiKek — APPROVED** [review]
LGTM

**Comment by MikeMcC399** [comment]
I don't see red text for "Delete Test" of a pending PCR test result.

(Also by the way, the confirmation dialog for deleting a PCR test waiting for a result says that it is deleting a result, even though there is not yet a result available.)

**Pull request merged**